tests/test_pandas_api.py

```diff
@@ -2029,3 +2029,136 @@ def test_keyed_loc_fixes(q):
         mkt[['k1', 'y']]
     with pytest.raises(KeyError):
         mkt['k1']
+
+
+def test_pandas_count(q):
+    tab = q('([] k1: 0n 2 0n 2 0n ; k2: (`a;`;`b;`;`c))')
+    df = tab.pd()
+
+    # Assert axis = 1
+    qcount = tab.count(axis=1).py()
+    pcount = df.count(axis=1)
+
+    print(pcount)
+    assert int(qcount[0]) == int(pcount[0])
+    assert int(qcount[1]) == 1
+
+    # Assert axis = 0
+    qcount = tab.count().py()
+    pcount = df.count()
+
+    assert int(qcount["k1"]) == int(pcount["k1"])
+    assert int(qcount["k2"]) == 3
+
+    # Assert only numeric
+    qcount = tab.count(numeric_only=True).py()
+    pcount = df.count(numeric_only=True)
+
+    assert int(qcount["k1"]) == int(pcount["k1"])
+
+
+def test_df_add_prefix(kx, q):
+    q('sym:`aaa`bbb`ccc')
+    t = q('([] 10?sym; til 10; 10?10; 10?1f)')
+
+    q_add_prefix = t.add_prefix("col_", axis=1)
+
+    assert(q('{x~y}', q_add_prefix, t.pd().add_prefix("col_")))
+
+    kt = kx.q('([idx:til 5] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_prefix = kt.add_prefix("col_", axis=1)
+    assert(q('{x~y}', q_add_prefix, kt.pd().add_prefix("col_")))
+
+    with pytest.raises(ValueError):
+        t.add_prefix("col_", axis=0)
+
+
+def test_df_add_suffix(kx, q):
+    q('sym:`aaa`bbb`ccc')
+    t = q('([] 10?sym; til 10; 10?10; 10?1f)')
+
+    q_add_suffix = t.add_suffix("_col", axis=1)
+
+    assert(q('{x~y}', q_add_suffix, t.pd().add_suffix("_col")))
+
+    kt = kx.q('([idx:til 5] til 5; 5?5; 5?1f; (5;5)#100?" ")')
+
+    q_add_suffix = kt.add_suffix("_col", axis=1)
+    assert(q('{x~y}', q_add_suffix, kt.pd().add_suffix("_col")))
+
+    with pytest.raises(ValueError):
+        t.add_suffix("_col", axis=0)
+
+
+def test_pandas_skew(q):
+    tab = q('([] price: 250.0f - 100?500.0f; ints: 100 - 100?200)')
+    df = tab.pd()
+
+    qskew = tab.skew().py()
+    pskew = df.skew()
+    assert round(float(qskew['price']), 6) == round(float(pskew['price']), 6)
+    assert round(float(qskew['ints']), 6) == round(float(pskew['ints']), 6)
+
+
+def test_std(kx, q):
+    df = pd.DataFrame(
+        {
+            'a': [1, 2, 2, 4],
+            'b': [1, 2, 6, 7],
+            'c': [7, 8, 9, 10],
+            'd': [7, 11, 14, 14]
+        }
+    )
+    tab = kx.toq(df)
+    p_m = df.std()
+    q_m = tab.std()
+    for c in q.key(q_m).py():
+        assert p_m[c] == q_m[c].py()
+    p_m = df.std(axis=1)
+    q_m = tab.std(axis=1)
+    for c in range(len(q.cols(tab))):
+        assert p_m[c] == q_m[q('{`$string x}', c)].py()
+    p_m = df.std(ddof=0)
+    q_m = tab.std(ddof=0)
+    for c in q.key(q_m).py():
+        assert p_m[c] == q_m[c].py()
+
+    p_m = df.std(ddof=4)
+    q_m = tab.std(ddof=4)
+    for c in q.key(q_m).py():
+        assert np.isnan(p_m[c]) == np.isnan(q_m[c].py())
+
+    q['tab'] = kx.toq(df)
+    tab = q('1!`idx xcols update idx: til count tab from tab')

```

Comment:
tab = q('`idx xkey update idx:i from tab')
```suggestion
    tab = q('`idx xkey update idx:i from tab')
```